examples/autopilot_private_firewalls/main.tf

```diff
@@ -33,7 +33,9 @@ provider "kubernetes" {
 }
 
 module "gke" {
-  source                            = "../../modules/beta-autopilot-private-cluster/"
+  source  = "terraform-google-modules/kubernetes-engine/google//modules/beta-autopilot-private-cluster/"
+  version = "~> 33.0"
```

Comment:
not sure why lint is complaining about this one, unless it was from the previous push?